langgraph/src/pregel/index.ts

```diff
@@ -384,10 +417,6 @@ export class Pregel
 
       if (stepOutput) {
         yield stepOutput;
-
-        if (typeof outputKeys !== "string") {
-          _applyWritesFromView(checkpoint, channels, stepOutput);
```

Comment:
Removing this had no effect (i.e. all tests passed). Need to double check if I need to migrate it.

Reply:
ah this is an api that used to exist in py but we removed, so good to remove here too